community/cypher/cypher/CHANGES.txt

```diff
@@ -1,5 +1,7 @@
 2.2.7
 -----
+o Setting dbms.cypher.statistics_divergence_threshold=1.0 disables replanning
+  due to non-schema changing updates.
```

Comment:
I would say `... disables replanning for all non-schema ...`. Also add to the aggregate changelogs.